app/src/androidTest/kotlin/dev/msfjarvis/aps/db/StoreDaoTest.kt

```diff
@@ -57,7 +58,7 @@ class StoreDaoTest {
   fun testInsertMultipleStores() {
     val storeEntries = arrayListOf<StoreEntity>()
     for (i in 0 until 7) {
-      storeEntries.add(StoreEntity(name = "store$i", external = false, initialized = false, isGitStore = false, uri = null))
+      val store = StoreEntity(0, "store", Uri.EMPTY, false, true, false)
```

Comment:
assigning to `store` seems like the wront thing to do here? it'll keep the `storeDao` empty and cause tests to fail

Reply:
I should really stop copying stuff. Fixing in a minute.